data/xml/2023.ijcnlp.xml

```diff
@@ -1490,7 +1482,20 @@
       <pages>77–85</pages>
       <url hash="12aca87c">2023.ijcnlp-demo.8</url>
       <bibkey>moon-etal-2023-wamp</bibkey>
-      <doi>10.18653/v1/2023.ijcnlp-demo.8</doi>
+    </paper>
+    <paper id="9">
+      <title><fixed-case>ERNIE</fixed-case>-Music: Text-to-Waveform Music Generation with Diffusion Models</title>
+      <author><first>Pengfei</first><last>Zhu</last></author>
+      <author><first>Chao</first><last>Pang</last></author>
+      <author><first>Yekun</first><last>Chai</last></author>
+      <author><first>Lei</first><last>Li</last></author>
+      <author><first>Shuohuan</first><last>Wang</last></author>
+      <author><first>Yu</first><last>Sun</last></author>
+      <author><first>Hao</first><last>Tian</last></author>
+      <author><first>Hua</first><last>Wu</last></author>
+      <pages>86–95</pages>
+      <url hash="645c502a">2023.ijcnlp-demo.9</url>
+      <bibkey>zhu-etal-2023-ernie</bibkey>
     </paper>
```

Comment:
It looks like this was reingested. This is a problem because the DOIs should be there. You can reinsert them manually, or run the `add_dois.py` script on just this volume (it just populates the XML, only if the DOI actually exists)